controller/Cargo.toml

```diff
@@ -32,6 +32,7 @@ lazy_static = "1"
 colored = "1.6"
 x25519-dalek = "0.6"
 ed25519-dalek = "1.0.0-pre.4"
+wagyu-ethereum = { git = "https://github.com/terryhash/wagyu-ethereum.git", branch = "master" }
```

Comment:
we will need move it to mwcproject repo

Reply:
How to clone it to mwcproject, will you or jim help?

Reply:
@jbyrer , can you help to clone?